drf/drf/settings.py

```diff
@@ -81,7 +83,7 @@
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'django-learning',
         'USER': 'postgres',
-        'PASSWORD': POSTGRE_PASSWORD,
+        'PASSWORD': POSTGRE_PASS,
```

Comment:
You don't need Posgres pass on every place. You could use ```PASSWORD: config('POSTGRE_PASS')```

Reply:
I did that on line 30